### DISCUSSION TITLE
探討成交量

### DISCUSSION THREAD

**Discussion by ChiuAlfredo** [ChiuAlfredo/options_day_data#24] [研究]
# 研究目標：結算日當天每一分鐘成交量變化率對不同價性選擇權分報酬率的影響

### 0. 研究資料與公式
* 結算日選擇權每一分鐘資料（買權成交量、買權成交價、賣權成交量、賣權成交價）

* 成交量變化率公式
$\delta V = ln(\frac{V_1}{V_0})$
$V_1$：該分鐘的成交量
$V_0$：該分鐘前一分鐘的成交量

* 報酬率公式
$R = ln(\frac{V_1}{V_0})$
$V_1$：該分鐘價格
$V_0$：該分鐘前一分鐘的價格

### 1. 研究價性
* 價內、價平、價外

#### 買權（call option）
$\frac{K}{S}$​ 範圍 | 價性分類
-- | --
$0.94 \leq \frac{K}{S} ​<0.97 $ | 深度價內 (DITM)
$0.97 \leq \frac{K}{S} ​<0.99 $ | 價內 (ITM)
$0.99\leq \frac{K}{S} <1.01 $ | 價平 (ATM)
$1.01\leq \frac{K}{S} ​<1.03 $ | 價外 (OTM)
$1.03 \leq \frac{K}{S} ​<1.06 $ | 深度價外 (DOTM)

#### 賣權（put option）
$\frac{K}{S}$​ 範圍 | 價性分類
-- | --
$1.03 \leq \frac{K}{S} ​<1.06 $ | 深度價內 (DITM)
$1.01\leq \frac{K}{S} ​<1.03 $ | 價內 (ITM)
$0.99\leq \frac{K}{S} <1.01 $ | 價平 (ATM)
$0.97 \leq \frac{K}{S} ​<0.99 $ | 價外 (OTM)
$0.94 \leq \frac{K}{S} ​<0.97$ | 深度價外 (DOTM)

* 將Call與Put分別進行分析

* 將成交量變化率與報酬率進行回歸分析

最後將產出以下六組與每日成交量變化率的迴歸分析結果：
深度價內Call
深度價內Put
價內Call
價內Put
價平Call
價平Put
價外Call
價外Put
深度價外Call
深度價外Put

### 2.後續分析
* 如整體相關性呈現高度相關(R Squared>0.8)，觀察其整體走勢並分析買權及賣權的權利金價格大幅漲跌時的成交量情形。

### 3.預期結果
* 觀測結算日成交量變化率與報酬率的相關性

* 確定成交量變化率在何種價性有最佳的報酬率

**Comment by ChiuAlfredo** [comment]
1. 時間為何
2. 何種價性
3. 我看到探討的目的是成交量波動但是研究方法是成交量?
4. 是否買賣權都要做比較
5. 或許可以考慮用t test去檢測兩者是否有顯著,或是使用迴歸去做說明
6. 相關性要用什麼公式
7. 研究方法2看不出來相關性跟gamma有什麼關聯

**Reply by TsuYiii** [reply]
3. 待確認問題方向

**Reply by TsuYiii** [reply]
已更新

**Comment by ChiuAlfredo** [comment]
1. 列出相關係數公式
2. 是結算日的每一分鐘還是有交易的每一分鐘
3. 怎麼定義價內價外價平
4. 研究價性中的為什麼會是每日變化率跟獲利率去跑迴歸
5. 變化率跟獲利率計算方式
6. 為什麼相關性低要去看交易量情況？

**Reply by TsuYiii** [reply]
已全數更正

**Comment by TsuYiii** [comment]
加上前一天總成交量、結算日當天每時段未平倉量

**Comment by TsuYiii** [comment]
價內綁在一起，同價位不分日期進行分組
用K/S去計算價平，一檔位包含兩三檔

**Comment by ChiuAlfredo** [comment]
* 成交量的變化率使用自然對數衡量 [探討股價跟成交量的相關性論文](https://www.pnas.org/doi/full/10.1073/pnas.0911983106)

**Comment by TsuYiii** [comment]
把所有價外、價平、價內各自取成一團，同一區不影響

**Comment by TsuYiii** [comment]
$r = \frac{n(\sum xy) - (\sum x)(\sum y)}{\sqrt{[n \sum x^2 - (\sum x)^2][n \sum y^2 - (\sum y)^2]}}$

**Comment by TsuYiii** [comment]
$V_t=ln(\frac{Q_t}{Q_{t-1}})$

**Comment by TsuYiii** [comment]
$r_t=ln(\frac{S_t}{S_{t-1}})$
